test/hdt-test.js

```diff
@@ -260,6 +261,60 @@ describe('hdt', function () {
         });
       });
 
+      describe('with pattern ex:s2 ex:p1 null', function () {
+        var triples;
+        before(function (done) {
+          document.searchTriples('http://example.org/s2', 'http://example.org/p1', null,
+                          function (error, t, c, e) { triples = t; done(error); });
+        });
+
+        it('should return an array with matches', function () {
+          triples.should.be.an.Array;
+          triples.should.have.lengthOf(10);
```

Comment:
Nitpick: I'm using `length()` throughout.

Reply:
@RubenVerborgh I found only one use of length() in the hdt-test.js and lengthOf() is used in all other cases. I changed all of them into length(). Is that what you meant? Thank you! :)